contracts/contracts/lib/LookupKey.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+library LookupKey {
+    /**
+     * @notice Returns the key used to lookup authorizations
+     * @param ritualId The ID of the ritual
+     * @param encryptor The address of the encryptor
+     * @return The key used to lookup authorizations
+     */
+    function lookupKey(uint32 ritualId, address encryptor) external pure returns (bytes32) {
```

Comment:
I'd suggest to have it `internal`, some time before there were many troubles with external/public functions in libraries. But if it's working well then no issue

Reply:
Setting `internal` here also fixed a library linking issue I had with `ape` :+1: